pommerman/graphics.py

```diff
@@ -183,7 +183,7 @@ def rgb_array(board, board_size, agents, is_partially_observable,
             for col in range(board_size):
                 value = board[row][col]
                 if utility.position_is_agent(board, (row, col)):
-                    num_agent = value - num_items
+                    num_agent = value - num_items + 4
```

Comment:
should this +4 be there in all situations?

Reply:
The former statement is based on this logic:

> The `value - num_items` can be the index of both agent in `agents` and agent value in `constant.Item`.

However, when there are only two `agents` in a game, this doesn't make sense. When `value` equals `10`, the `num_agent` equals `-4`, which will raise a `IndexError: list index out of range` because the length of `agents` is `2`. We can easily correct it by changing indices `-4, -3, -2, -1` to `0, 1, 2, 3`.

No matter how many agents in `agents`, the `x`-th (negative index) agent value in `constant.Item` corresponds to the `x + 4`-th agent's value in `agents`. Thus, I made this change. And it probably makes sense in all situations.